react-forms/src/TimePicker.tsx

```diff
@@ -32,11 +34,23 @@ export default function DatePicker<T>(props: {
   Disabled?: boolean;
   Feedback?: string;
   Step?: number;
+  Help?: string|JSX.Element;
 }) {
+  const [guid] = React.useState<string>(CreateGuid());
```

Comment:
Nitpicky but, I think having this as a Ref would be better just to be consistent, reason being that in https://github.com/GridProtectionAlliance/gpa-gemstone/blob/master/react-interactive/src/DropdownButton.tsx#L45 we also need a persistent GUID and we have it as a Ref.